integration-tests/test_proxy.py

```diff
@@ -0,0 +1,61 @@
+"""tests for the proxy"""
+import os
+import shutil
+import ssl
+from subprocess import Popen
+import time
+
+import requests
+
+from tljh.config import reload_component, set_config_value, CONFIG_FILE
+
+
+def test_manual_https(preserve_config):
+    ssl_dir = "/etc/tljh-ssl-test"
+    key = ssl_dir + "/ssl.key"
+    cert = ssl_dir + "/ssl.cert"
+    os.makedirs(ssl_dir, exist_ok=True)
+    os.chmod(ssl_dir, 0o600)
+    # generate key and cert
+    Popen(
```

Comment:
Why Popen rather than check_output? Since we aren't calling 'wait' explicitly here there's a race condition here too

Reply:
👍